datanode/sqlstore/migrations/0099_add_stop_order_rejection_reason.sql

```diff
@@ -0,0 +1,7 @@
+-- +goose Up
+
+ALTER TYPE stop_order_rejection_reason ADD VALUE IF NOT EXISTS 'REJECTION_REASON_STOP_ORDER_LINKED_PERCENTAGE_INVALID' AFTER 'REJECTION_REASON_STOP_ORDER_NOT_CLOSING_THE_POSITION';
```

Comment:
don't you want to be nice and also add it in schema.graphql? (only REJECTION_REASON_STOP_ORDER_LINKED_PERCENTAGE_INVALID is missing)

Reply:
didn't spot that